opsdroid/connector/matrix/connector.py

```diff
@@ -31,8 +31,6 @@ def __init__(self, config, opsdroid=None):  # noqa: D107
 
         self.name = "ConnectorMatrix"  # The name of your connector
         self.rooms = config.get("rooms", None)
-        if not self.rooms:
```

Comment:
I was looking at the PR and are we sure we need to remove this from the code? 
since we need a room called main, perhaps we should leave this in place?  What do you think @Cadair ?

Reply:
I think we should just not make rooms optional. See suggestion on the line above.

Reply:
Since this is already checked by the schema the optional rooms are not needed. Thanks for the suggestion, I just added it.